libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/connectingAttributesDropdown/ConnectingAttributesDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+// (C) 2007-2021 GoodData Corporation
+
+import { ObjRef } from "@gooddata/sdk-model";
+import React from "react";
+import { Dropdown } from "@gooddata/sdk-ui-kit";
+
+// const DROPDOWN_BODY_WIDTH = 205;
+
+export type HandleConnectingAttributeSelect = (ref: ObjRef, targetRef: ObjRef) => void;
+
+export interface IConnectingAttribute {
+    ref: ObjRef;
+    title: string;
+    icon: string;
+}
+
+interface IConnectingAttributesDropdownProps {
+    objRef: ObjRef;
+    selectedConnectingAttributeRef: ObjRef;
+    connectingAttributes: IConnectingAttribute[];
+    onSelect: HandleConnectingAttributeSelect;
+}
+
+// const getConnectingAttributeBySelection = (
+//     selectedConnectingAttributeRef: ObjRef,
+//     connectingAttributes: IConnectingAttribute[],
+// ): IConnectingAttribute => {
+//     return connectingAttributes.find(({ ref }) => areObjRefsEqual(ref, selectedConnectingAttributeRef))!;
+// };
+
+export const ConnectingAttributesDropdown: React.FC<IConnectingAttributesDropdownProps> = () =>
+    // {
+    //     objRef,
+    //     selectedConnectingAttributeRef,
+    //     connectingAttributes,
+    //     onSelect,
+    // },
+    {
+        // const selectedConnectingAttribute = getConnectingAttributeBySelection(
+        //     selectedConnectingAttributeRef,
+        //     connectingAttributes,
+        // );
+
+        return (
+            <Dropdown
+                className="connecting-attributes-dropdown s-connecting-attributes-dropdown"
+                closeOnParentScroll={true}
+                closeOnMouseDrag={true}
+                closeOnOutsideClick={true}
+                alignPoints={[
+                    {
+                        align: "br tr",
+                    },
+                    {
+                        align: "tr br",
+                    },
+                ]}
+                // button={
+                //     <DropdownButton
+                //         value={selectedConnectingAttribute.title}
+                //         iconLeft={selectedConnectingAttribute.icon}
+                //         title={selectedConnectingAttribute.title}
+                //     />
+                // }
+                //@ts-expect-error Functionality will be implemented in a future
```

Comment:
Why does this print an error in the first place?

Reply:
It expects a component with some props not provided at the moment.